shell/platform/darwin/ios/framework/Source/FlutterTextInputPluginTest.m

```diff
@@ -185,22 +226,22 @@ - (void)testSelectionChangeTriggersUpdateEditingClient {
   inputView.markedTextRange = nil;
   inputView.selectedTextRange = nil;
 
-  [inputView
+  BOOL shouldUpdate = [inputView
```

Comment:
These tests were reporting false negatives. `[FlutterTextInputView setTextInputState:]` now longer implicitly calls `updateEditingState` so none of them were supposed to be passing :(.